packages/theme/example/index.html

```diff
@@ -515,6 +515,95 @@ <h1 id="buttons">Buttons</h1>
               </div>
             </div>
 
+            <div id="btn-dropup-icons" style="margin-bottom: 15px;">
```

Comment:
If you add some markup, please don't forget to add `#btn-dropup-icons` selector to our screenshots file [here](https://github.com/Talend/ui/blob/master/packages/theme/screenshots.config.json)